### PR TITLE
Hw3

### PR DESCRIPTION
**Pull request by roxannefelig**
@whitneykasenetz @njie-DP

### CODE REVIEW

File: HW3/hw03.Rmd [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+  mutate_at(vars(comp1:student),
+         ~ifelse(. == -99, NA, .)
+  )
+
+summary(data) #now we see that the min values for variables are 1, not -99.
+
+melt_data<-melt(data) #collapses data to one column
+View(melt_data)
+
+#plotting using the melted data and the facet wrap will allow us to see histograms for each scale item, and the frequency of each response choice.
```
Comment (njie-DP) [inline HW3/hw03.Rmd:50]: I like how you review the distribution of each item.

File: HW3/hw03.Rmd [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+### 2. Compute total scores for the four scales. Recode variables as needed.
+
+#First, we need to reverse code items as indicated in the dictionary. The reverse coded items are green2,comp6:comp10, intel7:intel10,and open7:open10. 
+#two ways to reverse code shown below:
+
+reversed1<- c("green2", "comp6", "comp7","comp8","comp9","comp10","intel7",
+             "intel8","intel9","intel10","open7","open8","open9","open10")
+
+reversed2 <- dictionary %>%
+  filter(Keying == -1) %>%
```
Comment (njie-DP) [inline HW3/hw03.Rmd:70]: In my file, the key for green2 is -2, I'm not sure whether you have changed that in the file.

File: HW3/hw03.Rmd [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+  mutate(
+    .id = id,
+    green_total = mean(c_across(green1:green5), na.rm = TRUE),
+    comp_total = mean(c_across(comp1:comp10), na.rm = TRUE),
+    intel_total = mean(c_across(intel1:intel10), na.rm = TRUE),
+    open_total = mean(c_across(open1:open10), na.rm = TRUE),
+    .before = everything()
+  ) 
+
+summary(new_data)
```
Comment (njie-DP) [inline HW3/hw03.Rmd:97]: I used View(new_data) and found some items coded as "NaN" and some as "NA", you may want to double check with it.

File: HW3/hw03.Rmd (hunk 2 above)
Comment (roxannefelig) [reply HW3/hw03.Rmd:70]: He uploaded a new version after realizing that error, so my file did not have the -2

File: HW3/hw03.Rmd (hunk 3 above)
Comment (njie-DP) [reply HW3/hw03.Rmd:97]: I think that gave me an error when I try to run your code for question 3, maybe it's only on my end though.

**Review by njie-DP — COMMENTED** [review]
I think we reach similar interpretations to the dataset, but when I copied and run your codes on my end, there was an error. Other than that, your codes are clean and easy to follow!

File: HW3/hw03.Rmd (hunk 3 above)
Comment (roxannefelig) [reply HW3/hw03.Rmd:97]: huh, that's weird.. I ran into no errors and had no problem knitting. Do you remember which line the error was on?

Comment (njie-DP) [reply HW3/hw03.Rmd:97]: I tried to run it again today and there was no error lol.

**Pull request merged**